DigitalLearningSolutions.Data/Models/Progress.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace DigitalLearningSolutions.Data.Models
+{
+    using System;
+
+    public class Progress
+    {
+        public int ProgressId { get; set; }
+        public int CandidateId { get; set; }
+        public int CustomisationId { get; set; }
+        public DateTime? Completed { get; set; }
+        public DateTime? RemovedDate { get; set; }
+        public bool SystemRefreshed { get; set; }

```

Comment:
Looked back at some old conversations with Stella to find out. In the context of another ticket that was looking at progress records:

> The SystemRefreshed is to do with automatic removals, but when it's set, the RemovedDate is always set at the same time, so actually we don't need both

So I reckon it can safely be removed here as we now should only rely on RemovedDate.

Reply:
Yeah, Stella said that when the system refreshes the course, it both sets the removed date and changes `SystemRefreshed` from 0 to 1. (So `SystemRefreshed` actually means `HasBeenSystemRefreshed`). But yeah, I think the implication there is that we don't need it.